phpstan-baseline.neon

```diff
@@ -0,0 +1,26 @@
+parameters:
+	ignoreErrors:
+		-
+			message: "#^Property Ibexa\\\\Tests\\\\PHPStan\\\\Rules\\\\Fixtures\\\\RequireAbstractionInDependenciesFixture\\:\\:\\$abstractClass is never read, only written\\.$#"
```

Comment:
Maybe we can add ignores in the fixture classes to not glitter within the baseline? Those seem to be false-positives only. Exactly like you did in here: https://github.com/ibexa/phpstan/pull/6/files#diff-31c3739cf36481f21cd5681341982b8276bfd2c2a19a6e8fd2789e56e795948c.

Reply:
To be precise, if adding that to actual tested class, then analysed method checks for that as well, and we ends up with - for example
```
+22: No error with identifier property.onlyWritten is reported on line 22.
```